OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -266,7 +266,8 @@ public void showErrorDialog(String header, String message) {
     }
 
     @Override
-    public boolean createOptionsMenu(Menu menu) {
+    protected void onResume() {
+        super.onResume();
```

Comment:
I've played around with moving the actionbar setup to the oncreate method. The issue I am seeing here is that the actionbar is not available yet to some of the screens because https://github.com/edx/edx-app-android/blob/master/OpenEdXMobile/src/main/java/org/edx/mobile/base/FindCoursesBaseActivity.java#L54 needs to be called first. 

We have a tech debt item for us to switch over to the new Toolbars which will allow us more flexibility with the styles, but this is a bigger refactor. https://openedx.atlassian.net/browse/MA-2995.

I've committed some code closer to something we are looking for, but still feels a bit weird.

Thoughts?